src/test/scala/intent/matchers/ToContainAllOfTest.scala

```diff
@@ -0,0 +1,27 @@
+package intent.matchers
+
+import intent.{Stateless, TestSuite}
+import intent.helpers.Meta
+
+class ToContainAllOfTest extends TestSuite with Stateless with Meta with
+  "toContainAllOf" focused:
+    "for Map":
+      "of String -> Int":
+        "when negated":
+          "error is described properly also for multiple elements" in:
+            runExpectation(
+              expect(Map("one" -> 1, "two" -> 2)).not.toContainAllOf("one" -> 1, "two" -> 2),
```

Comment:
I'd like to have a test for the negation of `toContainAllOf` where:

* Actual is one->1, two->2
* Expected is one->1, three->3

In the positive case, this should be a failure, since actual doesn't contain three->3. Thus the negative case should be a success, I think. Does it work like that?

Reply:
Disagree. To **not** contain **all** of, should fail if >0 pairs are present in the Map.

_Still_, adding such a test case would be good, just with a slightly different result than what you propose ;-)

Reply:
Ok, new year, new thoughts =)

I revisited this and I agree with your view and will add the following tests:

```scala
"when partially matching":
  "error is described properly" in:
    runExpectation(
      expect(Map("one" -> 1, "two" -> 2)).toContainAllOf("one" -> 1, "three" -> 3),
      """Expected Map(...) to contain:
      |  "three" -> 3""".stripMargin)
  "and negating":
    "should pass since Map is not missing both pairs" in:
      expect(Map("one" -> 1, "two" -> 2)).not.toContainAllOf("one" -> 1, "three" -> 3)
```

Reply:
Fixed in a9a5f01